src/diffusers/models/unet_2d_condition.py

```diff
@@ -697,6 +697,13 @@ def _set_gradient_checkpointing(self, module, value=False):
         if isinstance(module, (CrossAttnDownBlock2D, DownBlock2D, CrossAttnUpBlock2D, UpBlock2D)):
             module.gradient_checkpointing = value
 
+    def fuse_lora(self):
+        self.apply(self._fuse_lora_apply)
+
+    def _fuse_lora_apply(self, module):
+        if hasattr(module, "_fuse_lora"):
+            module._fuse_lora()
+
```

Comment:
Shouldn't this go to 

https://github.com/huggingface/diffusers/blob/ea1fcc28a458739771f5112767f70d281511d2a2/src/diffusers/loaders.py#L185 

?